include/flashinfer/sampling.cuh

```diff
@@ -943,9 +943,15 @@ struct RenormTempStorage {
     float max_val;
     float min_val;
     union {
-      float value;
-      int count;
-      ValueCount<float> pair;
+      struct {
+        float value0, value1;
+      } values;
```

Comment:
You don't have to name this struct, just leave it as:
```
struct {
  float value0, value1;
};
```
and you can access them by `temp_storage.block_aggregate.value0`, `temp_storage.block_aggregate.value1`.

Or easier,
```
float values[2];
```